src/content/developers/docs/nodes-and-clients/client-diversity/index.md

```diff
@@ -51,6 +51,10 @@ The execution layer data were obtained from [Ethernodes](https://ethernodes.org)
 
 Up to date client diversity data for the consensus layer is now available at https://clientdiversity.org/.
 
+## Execution Layer
+
+Up until now, the conversation around client diversity has focused mainly on the consensus layer. However, the execution client [Geth](https://geth.ethereum.org) currently accounts for around 85% of all nodes. This is problematic for the same reasons as for consensus clients. For example, a bug in Geth affecting transaction-handling or constructing execution payloads could lead to bugged information being finalized by the consensus clients. Therefore, Ethereum would be healthier with a more even distribution of execution clients, ideally with no client representing more than 33% of the network.
```

Comment:
```suggestion
Until now, the conversation around client diversity has focused mainly on the consensus layer. However, the execution client [Geth](https://geth.ethereum.org) currently accounts for around 85% of all nodes. This percentage is problematic for the same reasons as for consensus clients. For example, a bug in Geth affecting transaction handling or constructing execution payloads could lead to consensus clients finalizing invalid transactions. Therefore, Ethereum would be healthier with a more even distribution of execution clients, ideally with no client representing more than 33% of the network.
```

Reply:
Wasn't sure if 'bugged information being finalized' meant transactions, I tried to make it clearer... Please LMK if I misinterpreted

Reply:
I'm really wary of "invalid transactions" because it suggests transactions that violate the EVM rules set-out in the whitepaper (e.g. minting new ether, emptying arbitrary accounts, etc). "Bugged" could mean missing, maliciously or accidentally re-ordered or censored, etc.

But it's not a great word, maybe "problematic" or similar?

Reply:
Went with 'problematic or bugged transactions' for the sake of pulling this in @jmcook1186. LMK if that isn't correct 😀